dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/transformers/pages/annotations/SinceKotlinTransformer.kt

```diff
@@ -155,6 +163,8 @@ public class SinceKotlinTransformer(
 
             val version = versions[sourceSet]
 
+            // the structure of custom tag content for SinceKotlin should be in sync
+            // with how DefaultPageCreator.contentForAllTypes reads it
```

Comment:
Also not that critical, but if the two parts must be in sync, it makes sense to have them in a single place. For instance,

```kotlin
// inside one class or file and next to each other, maybe inside SinceKotlinVersion

fun createSinceKotlinCustomTag(): CustomTagWrapper { ... }

fun extractSinceKotlinFromCustomTag(customTagWrapper: CustomTagWrapper): SinceKotlinVersion? {}
```

The benefit of having these two things next to each other is that you see clearly the correlation between the two, and if one changes - it'll be easier to find and remember to change the other. And you'll be able to reference it in the KDoc